modal/cls.py

```diff
@@ -244,7 +244,7 @@ class _Cls(_Object, type_prefix="cs"):
     _class_service_function: Optional[
         _Function
     ]  # The _Function serving *all* methods of the class, used for version >=v0.63
-    _method_functions: Dict[str, _Function]  # Placeholder _Functions for each method
+    _method_functions: Dict[str, _Function] = None  # Placeholder _Functions for each method
```

Comment:
A little ugly to be setting one default value in the middle of a bunch of type annotations